composer.json

```diff
@@ -38,7 +37,6 @@
         }
     },
     "scripts": {
-        "test": "phpunit",
```

Comment:
Why is this removed?

Reply:
because now need run 2 type PHPUnit 
1) only run PHPUnit
2) run PHPUnit and track code coverage

Reply:
Why not configure php to always run with coverage?

Reply:
@shadowhand we can do it but I don't see the point in every version of the PHP track

Reply:
Oh, right. Forgot we are testing many versions.